docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -10,16 +10,54 @@ device-mqtt features using an MQTT-broker.
 
 ![MQTT Overview](MQTT_Example_Overview.png)
 
-## Prepare the Custom Device Configuration 
+## Enabling Multi-Level Topics
```

Comment:
I think this section needs to be moved down to just before `Start EdgeX Foundry on Docker` for better context

Reply:
done